services/web3/getWeb3.js

```diff
@@ -41,6 +53,7 @@ async function loadWeb3Injected() {
   if (!alreadyInjected) throw new Error("Metamask cannot be found");
 
   web3 = new Web3(web3.currentProvider);
+  await getPermission();
```

Comment:
should check for the presence of window.ethereum before trying to access window.ethereum.enable()

and according to this https://medium.com/metamask/eip-1102-preparing-your-dapp-5027b2c9ed76
should be using new Web3(window.ethereum) instead of web3.currentProvider